sdk/translation/azure-ai-translation-text/tsp-location.yaml

```diff
@@ -0,0 +1,3 @@
+directory: specification/translation/Azure.AI.TextTranslation
+commit: 4637adf8aac277915fad37135dc4d11d34e166b9
```

Comment:
just to confirm, this is the same commit we are using in dotnet repository. And this file is being used.

Reply:
Yes.